tests/test_cli.py

```diff
@@ -131,6 +133,22 @@ def test_export_dump(tmpdir, data):
     assert len(os.listdir(str(dumpdir))) == 6
 
 
+@pytest.mark.parametrize('tile_size', [256, 512])
+def test_export_tile_size(tmpdir, data, tile_size):
```

Comment:
I originally had been looking at testing the `contents` directly within `test_mod.py`, but after a little more investigation it seemed that it would be a bit more involved than anticipated to verify the shape of tile as contained in the `bytearray` actually written to the mbtiles file. So I combined the tile size control with the dump command and then verified that the dumped images had the correct shape.